Chary/Assignments/Functions/1_vowels_uppercase.py

```diff
@@ -0,0 +1,20 @@
+#Write a program convert all the vowels to uppercase for the given input
+#string.
+
+def vowels_uppercase(x):
+    y=[]
+    vowels="aeiou"
+    for i in x:
+        if i in vowels:
+            z=chr(ord(i)-32)
+            y.append(z)
+        else:
+            y.append(i)
+        result="".join(y)    
```

Comment:
Please avoid the additional list i.e y = [] and join() function to complete this program

Reply:
fixed